src/settings/settings.ts

```diff
@@ -222,6 +222,14 @@ export function getXMLSettings(javaHome: string | undefined, logfile: string, ex
     enabled: workspace.getConfiguration('redhat.telemetry').get('enabled', false)
   };
 
+  // Check workspace trust
+  const isWorkspaceTrusted = (workspace as any).isTrusted;
+  if (!isWorkspaceTrusted) {
```

Comment:
If `workspace.isTrusted` does not exist in the API, i.e. in Theia, `isWorkspaceTrusted` becomes `undefined`. This makes it impossible to enable `resolveExternalEntities` in this case.

Reply:
The ! doesnt solve this? I should say it explicitly like you mentioned then?

Reply:
Yes, since `!undefined` is true and `!false` is also true